src/scene_map.cpp

```diff
@@ -190,7 +190,7 @@ void Scene_Map::Update() {
 	}
 
 	if (!Main_Data::game_player->IsMoving() || Game_Map::GetInterpreter().IsImmediateCall()) {
-		if (Game_Temp::menu_calling || Game_Temp::menu_calling_immediate) {
+		if (Main_Data::game_data.party_location.menu_calling || Game_Temp::menu_calling_immediate) {
```

Comment:
Maybe Main_Data::game_data.party_location could be exposed via functions in Game_Party?